qt/scientific_interfaces/Inelastic/Analysis/FitPlotView.h

```diff
@@ -52,14 +52,14 @@ class Splitter : public QSplitter {
   QIcon m_icon;
 };
 
-class MANTIDQT_INELASTIC_DLL IndirectFitPlotView : public API::MantidWidget, public IIndirectFitPlotView {
+class MANTIDQT_INELASTIC_DLL FitPlotView : public API::MantidWidget, public IFitPlotView {
   Q_OBJECT
 
 public:
-  IndirectFitPlotView(QWidget *parent = nullptr);
-  virtual ~IndirectFitPlotView();
+  FitPlotView(QWidget *parent = nullptr);
+  virtual ~FitPlotView();
```

Comment:
I guess `virtual` keyword is redundant here since FitPlotView is not further derived into subclasses.